snippets/6-rubric.md

```diff
@@ -4,7 +4,7 @@ Other helpful resources:
 
 * the [HRA definitions of research, service evaluation and audit](http://www.hra-decisiontools.org.uk/research/docs/DefiningResearchTable_Oct2017-1.pdf)
 * the [HRA “is it research” decision tool](http://www.hra-decisiontools.org.uk/research/)
-* if  you are still unsure, you can write to the HRA directly (please contact OpenSAFELY to discuss further)
+* if  you are still unsure, email the HRA directly on queries@hra.nhs.uk (please contact OpenSAFELY to discuss further if necessary)
```

Comment:
If you enclose the email address in `<` and `>`, then the Markdown parser will render it as a [mailto](https://en.wikipedia.org/wiki/Mailto).

```suggestion
* if  you are still unsure, email the HRA directly on <queries@hra.nhs.uk> (please contact OpenSAFELY to discuss further if necessary)
```